scripts/launch-xrd

```diff
@@ -289,10 +299,12 @@ if ! "$CONTAINER_EXE" --version &>/dev/null; then
 fi
 
 # Obtain the platform from the chosen image.
-if [[ $PLATFORM == "" ]]; then
-    if ! PLATFORM="$(_get_platform "$IMAGE")"; then
-        bad_usage
+if [[ ! $PLATFORM ]]; then
+    if ! $CONTAINER_EXE inspect "$IMAGE" &>/dev/null; then
+        echo "Specified image is not loaded, attempting to pull..."
```

Comment:
Print to stderr `>&2`

Reply:
The echo? Why? This is expected behaviour, right?

Reply:
I guess you can go either way on this, but to me it is noise/debug and not desired output. For example, the only desired or 'stdout' output I would ever expect something like `launch-xrd` to potentially print is the launched container's ID.
E.g. there is a non-zero possibility for `CTR_ID="$(launch-xrd foo --detach)" - but obviously not supported right now.

Reply:
If you want to be able to get only the ctr ID that's what a `--quiet` arg is used for - I think I'd lean towards using stdout here.

Reply:
Final decision?